tests/panels/test_sql.py

```diff
@@ -383,8 +384,6 @@ def test_insert_content(self):
         """
         list(User.objects.filter(username="café"))
         response = self.panel.process_request(self.request)
-        # ensure the panel does not have content yet.
-        self.assertNotIn("café", self.panel.content)
```

Comment:
This test didn't just check for the presence of the string but also our strings vs bytes handling, see https://github.com/jazzband/django-debug-toolbar/commit/476b5ce2a9d8261886b6f4a60fcbe7c62693d7de

I'm not sure if this is still being tested?